source-salesforce-native/source_salesforce_native/models.py

```diff
@@ -109,12 +110,12 @@ class SalesforceResourceConfigWithSchedule(ResourceConfigWithSchedule):
 
 
 def default_start_date():
-    return EARLIEST_VALID_DATE_IN_SALESFORCE
+    return EPOCH
 
 
 class EndpointConfig(BaseModel):
     start_date: AwareDatetime = Field(
-        description="UTC date and time in the format YYYY-MM-DDTHH:MM:SSZ. Any data generated before this date will not be replicated. If left blank, all data will be replicated.",
+        description="UTC date and time in the format YYYY-MM-DDTHH:MM:SSZ. Any data generated before this date will not be replicated. If left blank, the start date will be set to the epoch (1970-01-01T00:00:00Z).",
```

Comment:
Seems like this could be even more recent, but perhaps it is possible to backdate records in salesforce.